src/include/raft.h

```diff
@@ -131,6 +131,12 @@ enum raft_buf_set_nbuf
     RAFT_BS_LARGE_NBUF = TCP_MGR_NTHREADS,
 };
 
+struct raft_server_instance {
+    struct raft_instance *ri;
```

Comment:
In the niova repos, we've been prefixing the structure members w/ the acronym of the structure name.  This is done to enhance searchability throughout the repo since the result is typically a unique token. 

In this structure, we'd rename `ri` -> `rsi_ri`.